lib/rubies/game.rb

```diff
@@ -198,6 +187,25 @@ def game
       end
       byebye
     end
-  end
 
+    private
+
+    def check_answer(current, input, target)
```

Comment:
It's a bit unclear, but `input` contains the string `current` (which the user has to do in order to find the target value).  `eval(input)` it contains the `current` associated with the `current` in the `def play_round` method. `lambda { eval(input) }`  "stores" `current` to be evaluated later. Look at lines 157 and lines 166 in `game.rb` as that might help.